src/test/java/simpleCalculatorTestSuite/SimpleCalculatorTest.java

```diff
@@ -0,0 +1,102 @@
+package simpleCalculatorTestSuite;
+
+import java.io.IOException;
+
+import calculator.SimpleCalculator;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+
+public class SimpleCalculatorTest {
+
+    @Test
+    public void sumTest() throws Exception{
```

Comment:
Are you sure you want to throw an exception from the test method? As far as I understood from the sum method implementation logic, it's throwing exception, when the integer limit exceeds. If so, then it seems a part of the designed functionality instead of an unexpected issue, isn't it?

Reply:
For this test case, I call sum method with such arguments, for which the method should not throw any exception, it should return a valid int result, and if an exception occurs, then the test method should fail, hence I need to mention that it throws an exception, although the Exception class is too abstract, and I have replaced it with the concrete exception (IllegalArgumentException).

Reply:
`IllegalArgumentException ` is an Unchecked Exception. So, you do not have to add `throws` keyword to the method signature

Reply:
got it, changed everything as it should, thanks.